scripts/tools/zap/tests/inputs/all-clusters-app.zap

```diff
@@ -4209,16 +4209,6 @@
           "define": "GENERAL_DIAGNOSTICS_CLUSTER",
           "side": "server",
           "enabled": 1,
-          "commands": [
-            {
-              "name": "TestEventTrigger",
```

Comment:
This is pretty messed up, since the command is in fact an incoming command on the server side.  But this would match the actual all-clusters-app better.

Reply:
I think this was a manual edit. Unfortunately zap cannot recover from this using UI only nor can it detect it. I imagine we have some sql select that filters by "client" and as a result things get duplicated. Reviewing these things manually is very very hard.